src/views/Auth/Auth.test.js

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/react';
+import Auth from './Auth';
+
+test('Auth should display Header and AuthForm when user is not logged in', () => {
+  const { container } = render(<Auth setCurrentUser={null} />);
```

Comment:
Hmm looks like this is setting the property of the function `setCurrentUser` to `null`

Reply:
It is; I wrote this test to check for when no currentUser could be found, but I'm also interested in creating a snapshot test for logged-in users - would love your input!